src/GetDocumentInsider/Commands/GetDocumentCommandWorker.cs

```diff
@@ -121,6 +121,8 @@ public static bool TryProcess(GetDocumentCommandContext context, IServiceProvide
                     using (var outStream = File.Create(context.OutputPath))
                     {
                         stream.CopyTo(outStream);
+
+                        outStream.Flush();
```

Comment:
Yes, streams should be flushed when disposed of. But, this still seems to make the file more available.